core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -426,6 +426,14 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
                             : false;
                 }
 
+                if (!enableAllSecurityServices) {
+                    final String enableAllSecurityServicesFromProperties = properties
+                            .getProperty("quarkus.all-security-services.enable");
```

Comment:
Shouldn't it be `quarkus-internal`? Wondering if you will have a warning if you do that.

Reply:
The existing properties don't use the quarkus-internal name. I don't think these end up at runtime.